### ISSUE TITLE
`cargo upgrade` command is not exists

### ISSUE THREAD

**Issue by bitmacs** [kwrooijen/cargo.el#102]
As the pic says, in the latest Cargo, there is no `upgrade` subcommand. So maybe we need to remove the cargo-upgrade action to keep up with the Cargo, and make cargo.el more compact.
 
![image](https://user-images.githubusercontent.com/1202134/76946157-906a5d00-693e-11ea-8af8-f27dccb85d8c.png)
```
$ cargo --version
cargo 1.42.0 (86334295e 2020-01-31)
```

**Comment by kwrooijen** [comment]
Maybe we can add the `upgrade`, and mark `update` as obsolete? Since some people might still be on the older version?

**Comment by bitmacs** [comment]
> Maybe we can add the `upgrade`, and mark `update` as obsolete? Since some people might still be on the older version?

Yeah, sounds like a better solution by taking care of more end users. Also, to make the package more concise, besides marking as obsolete, can also tell users that useless action will be removed in the next two releases.
